### PR TITLE
Add google_cloud_pipeline_components_model_upload_predict_evaluate notebook

### PR DESCRIPTION
**Pull request by KevinBNaughton**

### CODE REVIEW

File: notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb [hunk 1]
```diff
@@ -0,0 +1,986 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:1]: <p>an <code>BatchPredictionJob -&gt; </code>a <code>BatchPredictionJob</code></p><p> </p><p>Evaluation <code>system.Metrics</code> Artifact. -&gt; evaluation <code>system.Metrics</code> artifact.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:1]: <p>there are duplicate pip installs: tensorflow, ....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:1]: <p>ModelBatcgPredictOp and ModelUploadOp need to be upgraded to v1.</p><p> </p><p>Ex.</p><p>from google_cloud_pipeline_components.v1.model import ModelUploadOp</p><p>from google_cloud_pipeline_components.v1.batch_predict_job import ModelBatchPredictOp&nbsp;</p><p> </p><p>Move the GCPC imports into the pipeline definition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:1]: <p>I suggest passing in the value of artifact_uri=MODEL_URI, as a Pipeline parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:1]: <p>wrong json file to remove -&gt; valuation_demo_pipeline.json</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (KevinBNaughton) [reply notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:1]: <p>I cannot, due to instance schema uri and prediction schema uri using the MODEL_URI. Pipeline parameter and string cannot be concatenated: "TypeError: unsupported operand type(s) for +: 'PipelineParam' and 'str'"</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**